sys/include/net/gnrc/lwmac/hdr.h

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright (C) 2015 Daniel Krebs
+ *               2016 INRIA
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     net_lwmac
+ * @{
+ *
+ * @file
+ * @brief       Header definition LWMAC
+ * @internal
+ * @author      Daniel Krebs <github@daniel-krebs.net>
+ * @author      Shuguo Zhuo  <shuguo.zhuo@inria.fr>
+ */
+
+#ifndef GNRC_LWMAC_HDR_H_
```

Comment:
`NET_GNRC_LWMAC_HDR_H` according to [coding conventions](https://github.com/RIOT-OS/RIOT/wiki/Coding-conventions#header-guards)